README.md

```diff
@@ -19,15 +20,51 @@
 
 ---
 
-Wn is a Python library for exploring information in wordnets. Install
-it from PyPI and download some data:
+Wn is a Python library for exploring information in wordnets. 
+
+## Installation
+
+Install it from PyPI using **pip** and download some data:
 
 ```console
 $ pip install wn
 $ python -m wn download oewn:2022  # the Open English WordNet 2022
 ```
 
-Then start exploring:
+Or, install using **conda**, from conda-forge channel:
+
+```sh
+# Method-1
+conda install -c conda-forge wn
+
+# Method-2
+# Using: conda install CHANNELNAME::PACKAGENAME
+conda install conda-forge::wn
+```
+
+> 🔔 Tip: **Speeding up Conda Installations**
+>
+> To speedup conda installations, alternatively, consider installing `mamba` 
+> from conda-forge channel and then replace all `conda` commands with `mamba` 
+> as a drop-in replacement.
+>
+> ```sh
+> conda install -c conda-forge mamba
+> mamba install -c conda-forge wn
+> ```
+
+For more helpful tips on conda, refer to this handy [Conda Cheatsheet ↗️][#conda-cheatsheet]. :fire:
+
+> ℹ️ The conda-forge recipe for this python library is maintained at the 
+> following conda-forge **feedstock** repository.
+>
+> - **Feedstock**: <https://github.com/conda-forge/wn-feedstock>
+
+[#conda-cheatsheet]: https://conda.io/projects/conda/en/latest/_downloads/a35958a2a7fa1e927e7dfb61ebcd69a9/conda-4.14.pdf
```

Comment:
This is helpful information, but it's also not directly relevant to Wn and doesn't belong here in the README. I would like new users to see the install commands and the "Getting Started" example in the same screen.

Reply:
Ok. I see your point. This may serve as a bit of distraction. Since the information is useful, let me put it under an expandable `<details>` block. 

If someone wants to see the details, can click on it and see the additional useful information.

Reply:
Useful or not, it's not relevant. By this I mean that Wn's README is not the place to advertise alternative utilities or to teach people how to use conda. People can look that up, and also I don't want to have to update the README when the utilities or instructions change.

The link to the feedstock _is_ relevant, but it is already at the top as a badge.

So while I appreciate your effort to provide helpful instructions here, I think the most I'd accept is linking the word **conda** to https://conda.io:

```
Or, install using [**conda**](https://conda.io), from conda-forge channel:
```

Reply:
> The link to the feedstock is relevant, but it is already at the top as a badge.

The badge point to the package on anaconda. Not the feedstock. This is common design for all packages. I would suggest you keep this information.

> So while I appreciate your effort to provide helpful instructions here, I think the most I'd accept is linking the word conda to [conda.io](https://conda.io/)

The documentation of conda is enormous. And to my experience, a lot of folks do not know about these tricks. I understand your view on "non-advertising" and "keeping it short". I will drop the cheatsheet and the mamba install options.

Reply:
I have updated the README.